### DISCUSSION TITLE
Q/A Remote assets from packages/auth

### DISCUSSION THREAD

**Discussion by doiya46** [callstack/super-app-showcase#60] [General]
Hi @jbroma , can you demo Remote assets in packages/auth

I try  to load an png/svg image to `SignInScreen.tsx` following remote-assets at https://github.com/callstack/repack/pull/331 but it didn't work.

**Comment by jbroma** [comment]
Hi @doiya46,

we plan to showcase the use of remote-assets in this repository. In the meantime, you can open an issue in RePack with your reproduction of the problem so I can help you get it working.
